index.html

```diff
@@ -0,0 +1,154 @@
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, intial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+		<link href="style.css" rel="stylesheet" type="text/CSS">
```

Comment:
it would be good to correct `text/CSS` to `text/css`, which is how it is always written

Reply:
Thanks! Done